tests/README.md

```diff
@@ -1,5 +1,9 @@
 To use these tests run the following:
 
+# The environment variables set by the pipeline need to be overwritten. The kubernetes of the pipeline automatically override the target host, so therefore need to unset them so you can deploy pods within the correct environment.
+
 ```
-ansible-playbook ./openshift-deployment-ansible/tests/all.yml --extra-vars "hostIP=<IP_address_of_Bastion_host>"
+for var in \$(export | grep KUB | awk '{ print \$2 }' | sed 's/\\=.*//g'); do unset \$var; done; source ./openstack_rc.sh ;
```

Comment:
Testing this on my Mac suggests that 
export | grep KUB | awk '{ print \$2 }'
actually gives me "-x" rather than the variable name.

I suggest the following would be better (and cut is probably faster than awk+sed)
env | grep KUB | cut -d= -f1

(instead of export | grep ... | awk... | sed...)